src/LiveSplit.Core/Server/CommandServer.cs

```diff
@@ -291,6 +296,58 @@ private void ProcessMessage(string message, IConnection clientConnection)
                 State.IsGameTimePaused = true;
                 break;
             }
+            case "getgamename":
+            {
+                response = State.Run.GameName.ToString();
+                break;
+            }
+            case "getcategoryname":
+            {
+                response = State.Run.CategoryName.ToString();
+                break;
+            }
+            case "getcategoryvariables":
+            {
+                RunMetadata md = State.Run.Metadata;
+
+                //Region
+                string region = null;
+                if (md.Region != null && !string.IsNullOrEmpty(md.Region.Abbreviation) && md.Game != null)
+                {
+                    region = md.Region.Abbreviation;
+                }
+                else if (!md.GameAvailable && !string.IsNullOrEmpty(md.RegionName))
+                {
+                    region = md.RegionName;
+                }
+
+                //Platform
+                string platform = null;
+                if (!string.IsNullOrEmpty(md.PlatformName) && (!md.GameAvailable || (md.Game != null)))
+                {
+                    platform = md.PlatformName;
+                }
+
+                //Variables
+                Dictionary<string, string> variables = [];
+                IEnumerable<string> variableL = md.VariableValueNames.Keys;
+                if (md.GameAvailable && md.Game != null)
+                {
+                    string categoryId = (md.CategoryAvailable && md.Category != null) ? md.Category.ID : null;
+                    variableL = md.Game.FullGameVariables.Where(x => x.CategoryID == null || x.CategoryID == categoryId).Select(x => x.Name);
```

Comment:
```suggestion
                    variableL = md.Game.FullGameVariables.Where(fgv => fgv.CategoryID == md.Category?.ID).Select(fgv => fgv.Name);
```

* `md.CategoryAvailable` and `md.Category != null` check the same thing. It's not possible for only one to be true.
* Use more fitting lambda parameter names. `x` is not acceptable.
* Do we really want to include variables where `CategoryID` is null?
* `fgv.CategoryID == md.Category?.ID` checks the full game variable's `CategoryID` against either `null` if `md.Category` was `null`, or `md.Category.ID` if it wasn't. It's simply more concise.